packages/compare-slider-block/src/CompareSliderBlock.tsx

```diff
@@ -97,6 +97,19 @@ export const CompareSliderBlock = ({ appBridge }: BlockProps) => {
     const secondAssetTitle = secondAsset ? (secondAssetAlt ?? '') : '';
     const secondAssetPreviewUrl = secondAsset ? secondAsset[0].previewUrl : '';
 
+    const aspectRatio = useMemo(() => {
+        if (!firstAsset || !secondAsset || height !== Height.Auto) {
+            return;
+        }
+
+        const assetWithSmallerAspectRatio =
```

Comment:
All this can probably be reduced to:
```suggestion
        const getRatio = ({ width, height }) => width / height
        return Math.min(getRatio(firstAsset[0], getRatio(secondAsset[0]))
```